applications/parami-protocol.md

```diff
@@ -146,29 +146,31 @@ Yanping YANG
 ## Development Roadmap :nut_and_bolt:
 
 ### Overview
-* **Total Estimated duration:** 4 months
+* **Total Estimated duration:** 6 months
 * **Full-time equivalent (FTE):** 3 FTE
 * **Total Costs:** 1 BTC
 
 ### Milestone 1 — The Parami Protocol
-* **Estimated duration:** 4 month
+* **Estimated duration:** 12 month
```

Comment:
Or 12 months?

Reply:
Hi @Noc2, sorry it’s 12 months. I have aligned the duration. We spent quite a lot time exploring Zero Knowledge Proof and its adoption to advertising at the beginning of development, including studying cryptography with researchers in academia, which extended the duration.